inst/extdata/water/elec_tech_water_map.csv

```diff
@@ -0,0 +1 @@
electricity,biomass,biomass (IGCC CCS),elec_biomass (IGCC CCS),biomass (IGCC CCS),biomass (IGCC CCS) (recirculating),IGCC (CCS),recirculating,fresh,electricity generation,biomass,biomass (IGCC CCS),regional biomass
electricity,biomass,biomass (IGCC CCS),elec_biomass (IGCC CCS),biomass (IGCC CCS),biomass (IGCC CCS) (dry cooling),IGCC (CCS),dry cooling,fresh,electricity generation,biomass,biomass (IGCC CCS),regional biomass
electricity,nuclear,Gen_II_LWR,elec_Gen_II_LWR,Gen_II_LWR,Gen_II_LWR (once through),nuclear,once through,fresh,electricity generation,nuclear,Gen_II_LWR,nuclearFuelGenII
electricity,nuclear,Gen_II_LWR,elec_Gen_II_LWR,Gen_II_LWR,Gen_II_LWR (seawater),nuclear,once through,seawater,electricity generation,nuclear,Gen_II_LWR,nuclearFuelGenII
electricity,nuclear,Gen_II_LWR,elec_Gen_II_LWR,Gen_II_LWR,Gen_II_LWR (recirculating),nuclear,recirculating,fresh,electricity generation,nuclear,Gen_II_LWR,nuclearFuelGenII
electricity,nuclear,Gen_II_LWR,elec_Gen_II_LWR,Gen_II_LWR,Gen_II_LWR (cooling pond),nuclear,cooling pond,fresh,electricity generation,nuclear,Gen_II_LWR,nuclearFuelGenII
electricity,nuclear,Gen_III,elec_Gen_III,Gen_III,Gen_III (once through),nuclear,once through,fresh,electricity generation,nuclear,Gen_III,nuclearFuelGenIII
electricity,nuclear,Gen_III,elec_Gen_III,Gen_III,Gen_III (seawater),nuclear,once through,seawater,electricity generation,nuclear,Gen_III,nuclearFuelGenIII
electricity,nuclear,Gen_III,elec_Gen_III,Gen_III,Gen_III (recirculating),nuclear,recirculating,fresh,electricity generation,nuclear,Gen_III,nuclearFuelGenIII
electricity,nuclear,Gen_III,elec_Gen_III,Gen_III,Gen_III (cooling pond),nuclear,cooling pond,fresh,electricity generation,nuclear,Gen_III,nuclearFuelGenIII
electricity,hydro,hydro,electricity,hydro,hydro,no cooling,none,fresh,electricity generation,hydro,hydro,
electricity,wind,wind_storage,electricity,wind,wind_storage,no cooling,none,none,electricity generation,wind,wind_storage,onshore wind resource
electricity,solar,PV_storage,electricity,solar,PV_storage,no cooling,none,fresh,electricity generation,solar PV,PV_storage,global solar resource
electricity,solar,CSP_storage,elec_CSP_storage,CSP_storage,CSP_storage (recirculating),CSP,recirculating,fresh,electricity generation,solar CSP,CSP_storage,global solar resource
electricity,solar,CSP_storage,elec_CSP_storage,CSP_storage,CSP_storage (dry_hybrid),CSP,dry_hybrid,fresh,electricity generation,solar CSP,CSP_storage,global solar resource
electricity,geothermal,geothermal,elec_geothermal,geothermal,geothermal (recirculating),geothermal,recirculating,fresh,electricity generation,geothermal,geothermal,geothermal
electricity,geothermal,geothermal,elec_geothermal,geothermal,geothermal (dry_hybrid),geothermal,dry_hybrid,fresh,electricity generation,geothermal,geothermal,geothermal
elect_td_bld,rooftop_pv,rooftop_pv,elect_td_bld,rooftop_pv,rooftop_pv,no cooling,none,none,,,,distributed_solar

```

Comment:
If your text editor will do that, then yes.  Otherwise, I can post a C program that you can compile and run to take care of it.